### ISSUE TITLE
dotnet build fails on linux with Service fabric actors enabled.

### ISSUE THREAD

**Issue by opsydev** [Microsoft/service-fabric-services-and-actors-dotnet#150]
**Describe the bug**
running dotnet build on official microsoft/dotnet docker images fails with 

```
/bin/sh: 3: /tmp/tmp2cb8afabe18d4ec4a35962d03ad7d1b8.exec.cmd: Syntax error: Unterminated quoted string
/root/.nuget/packages/microsoft.servicefabric.actors/3.2.176/build/netcoreapp2.0/Microsoft.ServiceFabric.Actors.targets(22,5): error MSB3073: The command "dotnet "obj/FabActUtilTemp\FabActUtil.dll" /spp:"PackageRoot" /t:manifest /sp:"PACKAGE_NAME" /in:"bin/Release/netcoreapp2.1/win7-x64//PACKAGE_NAME.dll" /arp:"/builds/PACKAGE_PATH/bin/Release/netcoreapp2.1/win7-x64/\" " exited with code 2. [/builds/PACKAGE_PATH/PACKAGE_NAME.csproj]
```

**To Reproduce**
Create a project with Service fabric actors as a dependency and run a dotnet build on the docker container.

**Expected behavior**
The build should pass.

**Additional context**
I think I have narrowed down the problem to this file

https://github.com/Microsoft/service-fabric-services-and-actors-dotnet/blob/develop/src/FabActUtil/Microsoft.ServiceFabric.Actors.targets

On line 21. There is an additional `\` in the `/arp` line.

**Other notes**
I am not sure why but the build passes just fine on powershell on windows. Also even though an error is raised dotnet build returns and exit code of 0 with build succeeded but shows "1 Error(s)". Any idea why that is?

**Comment by amanbha** [comment]
@opsydev Thanks for reporting, I will take a look at this. As a work around, to unblock your self: if you have application manifests and service manifests updated form a build on Windows for this project, you can use and set the property to UpdateServiceFabricManifestEnabled to false in .csproj for the Actor.

**Comment by opsydev** [comment]
I am more of a systems engineer on this project, so not super sure of what you mean there. :)

Our build process is all on linux docker containers. I'll forward the feedback to the appropriate people!

**Comment by amanbha** [comment]
@kentcb , yes, the config needs to be generated once before disabling the option as I mentioned in my comment above.

**Comment by olivergrimes** [comment]
Well this is excellent...  I can't build a fresh actor project - so I'll have to generate the config for myself!  
Wish me luck for the hours of debugging ahead.

**Comment by olivergrimes** [comment]
For anyone else finding themselves here... My issue seemed to be related to the many versions of .NET Core I had installed on my machine.  After I removed all .NET Core versions and re-installed VS, I could build actor projects.
